src/components/screens/delegates/voting/voteUrlProcessor/voteUrlProcessor.js

```diff
@@ -48,7 +48,13 @@ export default class VoteUrlProcessor extends React.Component {
   componentDidUpdate() {
     if (filterObjectPropsWithValue(this.state.voteLookupStatus, 'pending').length === 0
         && this.state.pendingVotes.length > 0) {
-      this.state.pendingVotes.map(v => this.props.voteToggled(v));
+      this.state.pendingVotes.forEach(v => this.props.voteToggled({
+        account: {
+          address: v.address,
+          publicKey: v.publicKey,
+        },
+        username: v.username,
```

Comment:
🐛 Now I noticed it needs to pass `rank` here. Else there is a regression and the page shows:

![Screenshot 2019-11-20 at 14 07 32](https://user-images.githubusercontent.com/1254342/69241455-2368d580-0b9f-11ea-904f-f8d3f2ce7121.png)

Instead of:
![Screenshot 2019-11-20 at 14 05 25](https://user-images.githubusercontent.com/1254342/69241428-14822300-0b9f-11ea-9e9d-2872136347bf.png)
```suggestion
        username: v.username,
        rank: v.rank,
```